content/blogposts/2022/ranking-in-a-week.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Ranking in a week'
+description: 'TODO'
+authors:
+  - name: Eric Fritz
+    url: https://eric-fritz.com
+publishDate: 2022-11-17T12:00+00:00
+tags: [blog]
+slug: ranking-in-a-week
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/TODO.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/TODO.png
+published: true
+---
+
+Last week we announced that we are now [ranking search results](https://about.sourcegraph.com/blog/new-search-ranking) on [https://sourcegraph.com](https://sourcegraph.com) in order to priotizie _relevant_ as well as _reusable_ code. We consider this Sourcegraph’s Code Intelligence Platform's first major victory of many, and a booming herald for a new era of Code Search.
+
+The effort to bring ranked results to our public instance was a concerted effort of four developers across two teams (Search and Code Intelligence), but took only a week of wall-clock time to design, implement, and deploy. The speed at which we were able to deliver this feature speaks volumes of our core data architecture.
+
+For the past three years at Sourcegraph, I have been amassing a giant compiler-accurate index of source code. And like a giant dork, that giant pile of data has just been sitting there enabling only the most basic of code navigation operations: go to definition and find references. The ranking effort is our first official foray into consuming this data in a more interesting way.
```

Comment:
This is hilarious, but is the dork you or the data?  If the former, then I'd suggest:

> And like a giant dork, I've let that data just sit there, enabling only the most basic...

But if the data is the dork then it's correct as-is.

Reply:
The data is the dork.